src/test/e2e/04_compare_scans_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2024 Defense Unicorns
+// SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+package test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var expectedTableHeaders = []string {
+	"ID", "Severity", "URL", "Advisory List",
+}
+
+func TestCompareScansCommand(t *testing.T) {
+	stdout, stderr, err := e2e.UDSPKDir("src/test", "compare-scans", "scans/alpine_3.16.json", "scans/alpine_3.17.json")
+	require.NoError(t, err, stdout, stderr)
+
+	expecteHeader := "### alpine:3.16 -> alpine:3.17"
+
+	expectedNew := "New vulnerabilities: 5"
+	expectedFixed := "Fixed vulnerabilities: 14"
+	expectedExisting := "Existing vulnerabilities: 4"
+
+
+	require.Contains(t, stdout, expecteHeader)
+	require.Contains(t, stdout, expectedNew)
+	require.Contains(t, stdout, expectedFixed)
+	require.Contains(t, stdout, expectedExisting)
```

Comment:
in cases like this it can be nice to use `assert` in place of `require` because you can see all failures instead of one at a time.

Reply:
fixed in next commit